Source/Scene/Batched3DModel3DTileContentProvider.js

```diff
@@ -740,7 +153,7 @@ define([
                 //
                 // We could also make another request for it, but that would make the property set/get
                 // API async, and would double the number of numbers in some cases.
-                that._batchTable = JSON.parse(batchTableString);
+                batchTable._batchTable = JSON.parse(batchTableString);
```

Comment:
Or maybe `Cesium3DTileBatchTable` could be renamed to `Cesium3DTileBatchResources`?